### DISCUSSION TITLE
%E6%99%AE%E5%8F%8A1/1.%20C%E8%AF%AD%E8%A8%80%E5%9F%BA%E7%A1%80%E5%BC%BA%E5%8C%96/%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B%E6%A0%BC%E5%BC%8F%E6%8E%A7%E5%88%B6

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#829] [Announcements]
# %E6%99%AE%E5%8F%8A1/1.%20C%E8%AF%AD%E8%A8%80%E5%9F%BA%E7%A1%80%E5%BC%BA%E5%8C%96/%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B%E6%A0%BC%E5%BC%8F%E6%8E%A7%E5%88%B6

image.pngimage.png 数据类型 整型（int、long long） image.png 有符号（默认） int（%d）：数据范围[−231，231−1]，即（-2147483648 ~ 2147483647）2∗109 long long（%lld）：数据范围[−263，263−1]，即 （-9223372036854775808 ~ ...

http://172.16.30.240:8080/%E6%99%AE%E5%8F%8A1/1.%20C%E8%AF%AD%E8%A8%80%E5%9F%BA%E7%A1%80%E5%BC%BA%E5%8C%96/%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B%E6%A0%BC%E5%BC%8F%E6%8E%A7%E5%88%B6.html

**Comment by Tiang-88** [comment]
6666
